### ISSUE TITLE
Line breaks, non continuous

### ISSUE THREAD

**Issue by gdp004** [xamarin/SignaturePad#18]
When using this component, the lines are not continuous. There is no flow.The attached image was me trying to move my finger around. Any ideas if I'm doing something wrong.
![img_7535](https://cloud.githubusercontent.com/assets/12418911/7599647/86bb1ca8-f8c9-11e4-9c5e-59d6bcc928a7.JPG)

**Comment by nosweat** [comment]
This is a dead project. no maintenance. I have no idea how to fix. I also get this exact result but what I have noticed is when I touch two fingers on the View, the first one is able to draw signature but the other finger should still touch the surface.

**Comment by nosweat** [comment]
@gdp004 Did you resolve this?
@Redth  What is the fix for this for iOS?

**Comment by nosweat** [comment]
I have solved this issue by tracing all the views/components that I have used on my Controller.

My SignaturePadView was contained in a Content Page that has a SidebarNavigation Component enabled.

Due to touch conflicts, I have simply changed my navigation controller to a normal Navigation Controller instead of using the SidebarNavigation Component with Nav Controller.

Whoever is using this component that encountered this same issue, I advise to put your signaturePadView on a View where there are no other touch interaction.
